website/src/components/dashboard/dashboard.tsx

```diff
@@ -94,30 +94,26 @@ export const BookmarksTab = () => {
   )
 }
 
-export const BookmarksTabItem = (props: { documentId: string }) => {
-  const [{ data: doc }] = useAnnotatedDocumentByIdQuery({
-    variables: { id: props.documentId },
-  })
-  const docData = doc?.documentByUuid
-  const docFullPath = docData?.chapters?.[0]?.path
+export const BookmarksTabItem = (props: { doc: DocumentFieldsFragment }) => {
+  const docFullPath = props.doc.chapters?.[0]?.path
```

Comment:
While we were migrating queries that relied on `AnnotatedDoc.audioRecording` it seemed nice to clean up this n+1 query.

Reply:
Tysm!